.buildkite/pipeline.yml

```diff
@@ -0,0 +1,34 @@
+steps:
+  - label: 'stack rebuild'
+    env:
+      AWS_REGION: us-west-1
+      S3_BUCKET: appveyor-ci-cache
+      CACHE_S3_MAX_SIZE: 2500MB
+      STACK_ROOT: "/build/cardano-node.stack"
+    command:
+      # cache-s3 needs a build directory that is the same across all buildkite agents.
```

Comment:
Yeah `cache-s3` required copying the sources into `/build/cardano-node` and building from there. This was so that the build directory would be the same across different build agents.

If not using `cache-s3` then there's no need to copy to `/build`.